modules/react-components/src/components/danger-zone/danger-zone-group.tsx

```diff
@@ -43,13 +47,15 @@ export const DangerZoneGroup: FunctionComponent<PropsWithChildren<DangerZoneGrou
 
     const {
         sectionHeader,
+        className,
         children,
         [ "data-componentid" ]: componentId,
         [ "data-testid" ]: testId
     } = props;
+    const defaultClassName = className ?? "danger-zone";
 
     return (
-        <div className="danger-zone-group-wrapper">
+        <div className= { (className ?? defaultClassName) + "-wrapper" }>
```

Comment:
this translates to className ?? className ?? "danger-zone", so we can just keep defaultClassName + "-wrapper", can't we?

Reply:
Resolved with e566b44176dd034f329c9cfb9c4a4d39093c7f56